circle.yml

```diff
@@ -5,6 +5,10 @@ general:
 machine:
   xcode:
     version: "7.3"
+  environment:
+    GOPATH: "$HOME/go"
+    PROJECT_ROOT: "$GOPATH/src/github.com/docker/hyperkit"
+    PATH: "${PATH}:${GOROOT}/bin:${GOPATH}/bin"
```

Comment:
Yes, I've never knowingly set `GOROOT` or used it for anything. Lets try without!